bin/node/runtime/src/lib.rs

```diff
@@ -119,7 +119,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 284,
+	spec_version: 286,
```

Comment:
Why 286 and not 285?

Reply:
I don't know the resaon why git show such diff.
In the [0.8.30 update](https://github.com/Cerebellum-Network/pos-network-node/pull/141/files#diff-67684005af418e25ff88c2ae5b520f0c040371f1d817e03a3652e76b9485224aR122) we `spec_version: 285`